core/src/main/java/com/linecorp/armeria/server/DefaultPathMappingContext.java

```diff
@@ -155,13 +155,9 @@ static PathMappingContext of(VirtualHost virtualHost, String hostname,
 
     @VisibleForTesting
     static MediaType resolveConsumeType(HttpHeaders headers) {
-        final String contentType = headers.get(HttpHeaderNames.CONTENT_TYPE);
+        final MediaType contentType = headers.contentType();
```

Comment:
Could we need to catch an exception from `headers.contentType()` not to be raised it to the caller?

Reply:
if ``DefaultHttpHeaders.contentType`` returns null is that acceptable?

Reply:
Yes. See https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/server/DefaultPathMappingContext.java#L152
`consumeType` passing into the constructor is `@Nullable`.